dodo.py

```diff
@@ -371,9 +419,6 @@ def task_venv():
             f'[ -f "{app_requirements}" ] && {PIP3} install -r "{app_requirements}"',
             f'[ -f "{test_requirements}" ] && {PIP3} install -r "{test_requirements}"',
         ],
-        'uptodate': [
-            lambda: os.environ.get('SKIP_VENV', None),
-        ],
     }
 
 def task_dynalite():
```

Comment:
TODO: Discuss the need of this if we instead use `serverless offline` for our local testing.